src/services/session-draft.ts

```diff
@@ -27,6 +27,30 @@ export class SessionDraft {
     this._orientations.push(correctedGammaWith90deg);
   }
 
```

Comment:
write a private function to calculate maxLeanAngel/gammaMax. The name of the function should be calculateGammaMax() with no parameter.